GitUI/CommandsDialogs/FormVerify.LostObject.cs

```diff
@@ -22,15 +22,40 @@ private enum LostObjectType
 
         private sealed class LostObject
         {
+            private static GitArgumentBuilder _logBuilder;
+            private static GitArgumentBuilder _tagBuilder;
+
             /// <summary>
             /// {0} - lost object's hash.
             /// %aN - committer name.
             /// %s  - subject.
             /// %ct - committer date, UNIX timestamp (easy to parse format).
             /// </summary>
-            private const string LogCommandArgumentsFormat = "log -n1 --pretty=format:\"%aN, %e, %s, %ct, %P\" {0}";
+            private static GitArgumentBuilder LogCommandArgumentsFormat
+            {
+                get
+                {
+                    _logBuilder = _logBuilder ?? new GitArgumentBuilder("log")
+                    {
+                        "-n1",
+                        "--pretty=format:\"%aN, %e, %s, %ct, %P\" {0}"
+                    };
+                    return _logBuilder;
+                }
+            }
 
-            private const string TagCommandArgumentsFormat = "cat-file -p {0}";
+            private static GitArgumentBuilder TagCommandArgumentsFormat
```

Comment:
Maybe something like:

```c#
private static readonly string TagCommandArgumentsFormat = new GitArgumentBuilder("cat-file") { "-p", "{0}" }.ToString();
```

There's no reason to hold the mutable builder that I can see. You can remove the field too. Same applies to `LogCommandArgumentsFormat`.

Reply:
Fixed in next push